### DISCUSSION TITLE
Links unavailable

### DISCUSSION THREAD

**Discussion by back0nline** [duhow/wibox#17] [Q&A] [answered]
Hi duhow

3 questions:
The linuch.pl links are no longer available, do you have any other place to get that info?

After flashing can I revert back the original firmware?

I see that is  uncheck "use intercom audio", what does that mean?

Thanks

**Comment by duhow** [comment, marked as answer]
- Use https://web.archive.org/ to view the previous stored content.
- This is a "patch" for a dumped image, as long as you keep the original image, you can restore it later. However I am also interested in retrieving the official update file, so if somebody gets it that can be appreciated to generate new images.
- Feature not available without using official app (Sofia).
